Backend/dev/migrations/dynamic-offer-driver-app/0165-reuse-old-estimate-in-repetition.sql

```diff
@@ -0,0 +1,135 @@
+  id,
+  transaction_id,
+  provider_id,
+  from_location_id,
+  to_location_id,
+  bap_id,
+  bap_uri,
+  estimated_distance,
+  estimated_duration,
+  auto_assign_enabled,
+  customer_language,
+  device,
+  created_at)
+  (SELECT * FROM SearchRequests);
+
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ALTER COLUMN search_request_id DROP NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ADD COLUMN request_id character(36);
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ADD COLUMN search_try_id character(36);
+UPDATE atlas_driver_offer_bpp.driver_quote SET request_id = search_request_id;
+UPDATE atlas_driver_offer_bpp.driver_quote SET search_try_id = search_request_id;
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ADD CONSTRAINT
+  driver_quote_to_search_request_fk FOREIGN KEY (request_id) REFERENCES atlas_driver_offer_bpp.search_request_2 (id);
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ADD CONSTRAINT
+  driver_quote_to_search_try_fk FOREIGN KEY (search_try_id) REFERENCES atlas_driver_offer_bpp.search_try (id);
+
+ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ALTER COLUMN search_request_id DROP NOT NULL;
+ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ADD COLUMN request_id character(36);
+ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ADD COLUMN search_try_id character(36);
+UPDATE atlas_driver_offer_bpp.search_request_for_driver SET request_id = search_request_id;
+UPDATE atlas_driver_offer_bpp.search_request_for_driver SET search_try_id = search_request_id;
+ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ADD CONSTRAINT
+  search_request_for_driver_to_search_request_fk FOREIGN KEY (request_id) REFERENCES atlas_driver_offer_bpp.search_request_2 (id);
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ADD CONSTRAINT
+  search_request_for_driver_to_search_try_fk FOREIGN KEY (search_try_id) REFERENCES atlas_driver_offer_bpp.search_try (id);
+
+ALTER TABLE atlas_driver_offer_bpp.search_try ADD COLUMN request_id character(36);
+UPDATE atlas_driver_offer_bpp.search_try SET request_id = id;
+ALTER TABLE atlas_driver_offer_bpp.search_try ADD CONSTRAINT
+  search_step_to_search_request_fk FOREIGN KEY (request_id) REFERENCES atlas_driver_offer_bpp.search_request_2 (id);
+
+ALTER TABLE atlas_driver_offer_bpp.search_request_for_driver ALTER COLUMN transaction_id DROP NOT NULL;
+
+ALTER TABLE atlas_driver_offer_bpp.estimate ALTER COLUMN transaction_id DROP NOT NULL;
+
+ALTER TABLE atlas_driver_offer_bpp.estimate ADD COLUMN request_id character(36);
+UPDATE atlas_driver_offer_bpp.estimate AS T1 SET request_id = (SELECT T2.id FROM atlas_driver_offer_bpp.search_try AS T2 WHERE T2.estimate_id = T1.id);
+ALTER TABLE atlas_driver_offer_bpp.estimate ADD CONSTRAINT
+  estimate_to_search_request_fk FOREIGN KEY (request_id) REFERENCES atlas_driver_offer_bpp.search_request_2 (id);
+
+ALTER TABLE atlas_driver_offer_bpp.driver_quote ALTER COLUMN transaction_id DROP NOT NULL;
```

Comment:
If we're dropping `transaction_id` from driver_quote, estimate, and search_request_for_driver table then where we will store the bap search request id in these table?

Reply:
> If we're dropping `transaction_id` from driver_quote, estimate, and search_request_for_driver table then where we will store the bap search request id in these table?

in SearchRequest